parachain/pallets/inbound-queue/src/lib.rs

```diff
@@ -180,6 +186,8 @@ pub mod pallet {
 		Send(SendError),
 		/// Message conversion error
 		ConvertMessage(ConvertMessageError),
+		/// Burn fees failure
+		BurnFees,
```

Comment:
```suggestion
```

Actually we should just raise [TokenError::FundsUnavailable](https://paritytech.github.io/polkadot-sdk/master/sp_runtime/enum.TokenError.html), which I believe is convertible to `DispatchError`.

Reply:
https://github.com/Snowfork/snowbridge/pull/1071/commits/85f5b42b97aa07e825b221a82c9079808a3909ef